imagedephi/base_rules.yaml

```diff
@@ -82,3 +82,615 @@ rules:
+      method: delete
+      type: metadata
+    - tag_name: CFAPattern
+      method: delete
+      type: metadata
+    - tag_name: CustomRendered
+      method: delete
+      type: metadata
+    - tag_name: ExposureMode
+      method: delete
+      type: metadata
+    - tag_name: WhiteBalance
+      method: delete
+      type: metadata
+    - tag_name: FocalLengthIn35mmFilm
+      method: delete
+      type: metadata
+    - tag_name: SceneCaptureType
+      method: delete
+      type: metadata
+    - tag_name: GainControl
+      method: delete
+      type: metadata
+    - tag_name: Contrast
+      method: delete
+      type: metadata
+    - tag_name: Saturation
+      method: delete
+      type: metadata
+    - tag_name: Sharpness
+      method: delete
+      type: metadata
+    - tag_name: SubjectDistanceRange
+      method: delete
+      type: metadata
+    - tag_name: GDAL_Metadata
+      method: delete
+      type: metadata
+    - tag_name: GDAL_NoData
+      method: delete
+      type: metadata
+    - tag_name: GPSLongitude
+      method: delete
+      type: metadata
+    - tag_name: '50838'
+      method: delete
+      type: metadata
+    - tag_name: '50839'
+      method: delete
+      type: metadata
+    - tag_name: '51123'
+      method: delete
+      type: metadata
+    - tag_name: '51168'
+      method: delete
+      type: metadata
+    - tag_name: JPEGProc
+      method: delete
+      type: metadata
+    - tag_name: JPEGIFOffset
+      method: delete
+      type: metadata
+    - tag_name: JPEGIFByteCount
+      method: delete
+      type: metadata
+    - tag_name: JPEGRestartInterval
+      method: delete
+      type: metadata
+    - tag_name: JPEGQTables
+      method: delete
+      type: metadata
+    - tag_name: JPEGDCTables
+      method: delete
+      type: metadata
+    - tag_name: JPEGACTables
+      method: delete
+      type: metadata
+    - tag_name: YCbCrCoefficients
+      method: delete
+      type: metadata
+    - tag_name: YCbCrPositioning
+      method: delete
+      type: metadata
+    - tag_name: '55000'
+      method: delete
+      type: metadata
+    - tag_name: '55001'
+      method: delete
+      type: metadata
+    - tag_name: '55002'
+      method: delete
+      type: metadata
+    - tag_name: '55003'
+      method: delete
+      type: metadata
+    - tag_name: '55004'
+      method: delete
+      type: metadata
+    - tag_name: '59932'
+      method: delete
+      type: metadata
+    - tag_name: Padding
+      method: delete
+      type: metadata
+    - tag_name: OffsetSchema
+      method: delete
+      type: metadata
+    - tag_name: GPSAltitudeRef
+      method: delete
+      type: metadata
+    - tag_name: '60010'
+      method: delete
+      type: metadata
+    - tag_name: '60011'
+      method: delete
+      type: metadata
+    - tag_name: '60020'
+      method: delete
+      type: metadata
+    - tag_name: '65200'
+      method: delete
+      type: metadata
+    - tag_name: NDPI_ByteCountHighBytes
```

Comment:
We shouldn't include ANY NDPI tags in our base rule set.  Sadly, NDPI files are *nearly* tiff files, but they aren't tiff files and handling them as tiff files will produce broken hdmi files.